src/system-health/health_checker/hardware_checker.py

```diff
@@ -35,27 +42,32 @@ def _check_asic_status(self, config):
         if config.ignore_devices and 'asic' in config.ignore_devices:
             return
 
-        temperature = self._db.get(self._db.STATE_DB, HardwareChecker.ASIC_TEMPERATURE_KEY, 'temperature')
-        temperature_threshold = self._db.get(self._db.STATE_DB, HardwareChecker.ASIC_TEMPERATURE_KEY, 'high_threshold')
-        if not temperature:
-            self.set_object_not_ok('ASIC', 'ASIC', 'Failed to get ASIC temperature')
-        elif not temperature_threshold:
-            self.set_object_not_ok('ASIC', 'ASIC', 'Failed to get ASIC temperature threshold')
-        else:
-            try:
-                temperature = float(temperature)
-                temperature_threshold = float(temperature_threshold)
-                if temperature > temperature_threshold:
-                    self.set_object_not_ok('ASIC', 'ASIC',
-                                           'ASIC temperature is too hot, temperature={}, threshold={}'.format(
-                                               temperature,
-                                               temperature_threshold))
-                else:
-                    self.set_object_ok('ASIC', 'ASIC')
-            except ValueError as e:
-                self.set_object_not_ok('ASIC', 'ASIC',
-                                       'Invalid ASIC temperature data, temperature={}, threshold={}'.format(temperature,
-                                                                                                            temperature_threshold))
+        for i in range(0, len(HardwareChecker.ASIC_TEMPERATURE_KEY)):
+            temperature = self._db.get(self._db.STATE_DB, HardwareChecker.ASIC_TEMPERATURE_KEY[i],
+                                                          'temperature')
+            temperature_threshold = self._db.get(self._db.STATE_DB, HardwareChecker.ASIC_TEMPERATURE_KEY[i],
+                                                          'high_threshold')
+            asic_name = HardwareChecker.ASIC_TEMPERATURE_KEY[i][17:]
```

Comment:
number 17 here is confusing, please avoid using magic number.

For example:
```python
asic_name = HardwareChecker.ASIC_TEMPERATURE_KEY[i].split('|')[1]
```

Reply:
Agreed.